src/components/common/HideTextInput/HideTextInput.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { string, func } from "prop-types";
+import conditionalClassName from "classnames";
+
+import visibility from "assets/icons/visibility.png";
+import visibilityOff from "assets/icons/visibility-off.png";
+
+import "./hide-text-input.scss";
+
+const HideTextInput = ({ name, label, placeHolder, value, handleChange }) => {
+  const [showText, setShowText] = useState(false);
+
+  const visibilitySrc = showText ? visibility : visibilityOff;
+
+  return (
+    <label className="regular-text">
+      {label}
+      <div className="hide-text-input-container">
+        <input
```

Comment:
I'd suggest adding the `htmlFor` attribute here to ensure the label is asociated to the input properly.
Also, I think it's best to keep the container div as a parent element to the label and input.
WDYT?